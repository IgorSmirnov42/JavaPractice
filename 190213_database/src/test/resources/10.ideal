Write command name (8 for help): Write owner name: Write phone number: NotNull annotated argument 1 of ru/spbhse/smirnov/database/PhoneDatabase.addRecord must not be null
Write command name (8 for help): 