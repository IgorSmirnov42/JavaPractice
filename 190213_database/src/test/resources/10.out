Write command name (8 for help): Write owner name: Write phone number: Argument for @NotNull parameter 'phoneNumber' of ru/spbhse/smirnov/database/PhoneDatabase.addRecord must not be null
Write command name (8 for help): 